src/libs/actions/User.js

```diff
@@ -837,6 +837,29 @@ function setContactMethodAsDefault(newDefaultContactMethod) {
     Navigation.navigate(ROUTES.SETTINGS_CONTACT_METHODS);
 }
 
+/**
+ * @param {String} theme
+ */
+function updateTheme(theme) {
+    const optimisticData = [
+        {
+            onyxMethod: Onyx.METHOD.SET,
+            key: ONYXKEYS.PREFERRED_THEME,
+            value: theme,
+        },
+    ];
+
+    API.write(
+        'UpdateTheme',
+        {
+            value: theme,
+        },
+        {optimisticData},
+    );
+
+    Navigation.navigate(ROUTES.SETTINGS_PREFERENCES);
+}
```

Comment:
@bernhardoj the `ThemePage` should now be available :)

Reply:
https://github.com/Expensify/App/blob/main/src/pages/settings/Preferences/ThemePage.js

Reply:
ah yea, makes sense.[This is the PR](https://github.com/Expensify/App/pull/21669) that's gonna add the button, but it's gonna take some more time until we merge it, because we have to migrate the whole app to theme switching first